### DISCUSSION TITLE
Can a snap have a link which when clicked opens the site in a new tab?

### DISCUSSION THREAD

**Discussion by javaspeak** [MetaMask/snaps#1804] [Q&A]
Can a snap have a link which when clicked opens the site in a new tab?

**Comment by Montoya** [comment]
We are adding links to custom UI. Right now it is not possible to use links but soon it will be possible.

**Reply by databu** [reply]
Any idea about the timeframe for this?

**Reply by FrederikBolding** [reply]
A couple of weeks probably, it is already in progress!
